runtime/jcl/common/mgmtthread.c

```diff
@@ -1344,6 +1359,18 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 
 	/* this may block on vm->managementDataLock */
 	getContentionStats(currentThread, targetThread, info);
+#if JAVA_SPEC_VERSION >= 19
+	/* getStackFramePCs need to run on the carrier thread's stack to produce the right output.
```

Comment:
`getStackFramePCs` is a singular function name: `it needs` vs `they need`.
```suggestion
	/* getStackFramePCs needs to run on the carrier thread's stack to produce the right output.
```